charts/judge-web/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
```

Comment:
We should have a service account for the entire deployment (assuming it's all running in the same namespace).